puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_1_17_stable.yml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: test_1_17_stable
+    project-type: matrix
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 32
+    quiet-period: 2700
+    properties:
+      - github_foreman
+    scm:
+      - foreman:
+          branch: '1.17-stable'
+    triggers:
+      - scm_fifteen_minutes
+      - github
+      - schedule_failed_builds
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+            - 2.2
```

Comment:
I think we can assume Ruby 2.3+ since Debian Stretch and Ubuntu Xenial are both 2.3 and our RH uses the 2.4 SCL.